css-basics/css-basics_pokemon-page/css/styles.css

```diff
@@ -4,23 +4,28 @@
 
 /* Task 1: First of all we want to adjust how the size of our elements is calculated when working with CSS. Do you know how to switch from "content-box" to "border-box"? */
 
-/* your solution */
-
 /* Task 2: Let's work on some basic stylings that we want to apply to all elements of your code via inheritance. Which selector do we need for this? */
 
-/* add selector */
- {
-  font-family: monospace;
-  font-size: 1rem;
-  line-height: 1.5;
-  background-color: LemonChiffon;
-}
+  * {
+    box-sizing: border-box;
+    margin: 0px;
+    padding: 0px;
+    font-family: monospace;
+    font-size: 1rem;
+    line-height: 1.5;
+    background-color: LemonChiffon;
```

Comment:
Colors are not typically camelCased in css. I checked in the browser and it seems to work but you should stick to lowercase maybe. Sorry I didn't spot this during my initial review.